src/mainwin.cc

```diff
@@ -732,8 +733,9 @@ void MainWindow::updateReorderMode(bool reorderMode)
 
 MainWindow::~MainWindow()
 {
-	if (root_module) delete root_module;
-	if (root_node) delete root_node;
+	delete root_module;
+	delete last_good_module;
```

Comment:
Yes that was a change you made in minor style tweaks, not me. It is fixed in my latest version. last_good_module is renamed parsed_module and it is deleted but root_node isn't because if it is non null it is guaranteed to be the same.